src/pages/s/index.tsx

```diff
@@ -1,8 +1,36 @@
 import { Grid, GridItem, GridProps, Heading, Text, VStack } from '@chakra-ui/react';
-import { componentSpacing, paragraphSpacing } from 'theme/metrics';
+import { componentSpacing, paragraphSpacing, partnerSpacing } from 'theme/metrics';
 import Image from "next/image";
 import wechatAppQrcode from '../../../public/img/wechat-app-qrcode.jpg';
+import partner_1 from '../../../public/img/partner_1.png';
+import partner_2 from '../../../public/img/partner_2.png';
+import partner_3 from '../../../public/img/partner_3.jpeg';
+import partner_4 from '../../../public/img/partner_4.png';
+import partner_5 from '../../../public/img/partner_5.png';
 import { sidebarBreakpoint } from 'components/Navbars';
+import { StaticImageData } from "next/image";
+
+const partnersData = [{
```

Comment:
what I meant in this comment https://github.com/yuanjian-org/app/pull/444#discussion_r1817695712 is to move this data structure into the PartnersSection function. also change its name to simply `partners`.